schedule/virt_autotest/slem_ppc_slem_installation_virt.yaml

```diff
@@ -0,0 +1,12 @@
+name:           slem_installation_virt
+description:    >
+    Maintainer: xguo@suse.com, qe-virt@suse.de
+    SLE Micro as virtualization host. It runs qemu as emulator and uses default selfinstall raw image.
+schedule:
+    - microos/disk_boot
+    - transactional/host_config
+    - console/suseconnect_scc
+    - qemu/info
+    - qemu/qemu
```

Comment:
@guoxuguang Hi Leon, would you please add a comment why kvm test module is not enabled now?

Reply:
Supper, had updated comments now.